src/Week03/Fun.hs

```diff
@@ -0,0 +1,88 @@
+module Week03.Fun () where
```

Comment:
This module seems to be unused (not even present in cabal fiel). Can we just remove it?